evm-contracts/src/v0.7/dev/Owned.sol

```diff
@@ -6,8 +6,8 @@ pragma solidity ^0.7.0;
  */
 contract Owned {
 
-  address public owner;
-  address private pendingOwner;
+  address public s_owner;
```

Comment:
Let's make this private and add a getter without the storage namespace on the variable.